device/flasher.go

```diff
@@ -69,7 +68,7 @@ func (d *flasher) DownloadImage() (fileName, filePath string, err error) {
 }
 
 // PrepareForFlashing method inits virtualbox, download necessary files from the repo into the vbox
-func (d *flasher) PrepareForFlashing() error {
+func (d *flasher) Prepare() error {
```

Comment:
__[gometalinter]__
* [gocyclo] cyclomatic complexity 33 of function (*flasher).Prepare() is high (> 15)

<!-- Comment by [SideCI](https://sideci.com) -->